core/src/test/java/kafka/server/share/SharePartitionTest.java

```diff
@@ -771,24 +768,20 @@ public void testMaybeInitializeWithReadException() {
         Persister persister = Mockito.mock(Persister.class);
         // Complete the future exceptionally for read state.
         Mockito.when(persister.readState(Mockito.any())).thenReturn(FutureUtils.failedFuture(new RuntimeException("Read exception")));
-        SharePartition sharePartition = SharePartitionBuilder.builder().withPersister(persister).build();
+        SharePartition sharePartition1 = SharePartitionBuilder.builder().withPersister(persister).build();
```

Comment:
nitL why do you need change in varibale names?

Reply:
I find the code cleaner this way since the 2 instances of `SharePartition` behave differently, hence the name change.